asv/commands/preview.py

```diff
@@ -84,9 +83,9 @@ def run_from_conf_args(cls, conf, args):
     def run(cls, conf, port=0, browser=False):
         os.chdir(conf.html_dir)
 
-        class Handler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+        class Handler(http.server.SimpleHTTPRequestHandler):
```

Comment:
Looks like `http.server` is not defined. We should import it.

@dorothykiz1 looks like we don't have tests for preview. It'd be good to create an issue for it. And also, if you want to have a look at other commands that are not tested, that would be very helpful. Not having any test for a command means that like here, even a bug like this one can be added unnoticed (this would fail as soon as the command is executed, for all cases).

Reply:
Thanks @datapythonista i will take a look at this